### DISCUSSION TITLE
Adding Lottie animated icons in icon layer.

### DISCUSSION THREAD

**Discussion by AhsanAshfaq1** [visgl/deck.gl#7262] [Q&A] [answered]
Hi everyone! I want to add Lottie icons as markers in icon layer. Is there any way I can do that. If not, then please tell me the appropriate way to add animated icons (i.e., gifs) as markers in deck.gl layer. Your help will be appreciated.

**Comment by Pessimistress** [comment, marked as answer]
https://codesandbox.io/embed/deck-gl-iconlayer-animation-m7ikj

**Reply by AhsanAshfaq1** [reply]
WOW! Thank you, kind sir. Thank you soo much.
